core/vm/actor/codes.hpp

```diff
@@ -58,3 +58,23 @@ namespace fc::vm::actor::code {
   constexpr Code system4{"fil/4/system"};
   constexpr Code verifreg4{"fil/4/verifiedregistry"};
 }  // namespace fc::vm::actor::code
+
+#define VM_ACTOR_BUILTIN_V_CODE(V)                             \
+  namespace fc::vm::actor::builtin::v##V {                     \
+    constexpr auto kAccountCodeId{code::account##V};           \
+    constexpr auto kCronCodeId{code::cron##V};                 \
+    constexpr auto kStoragePowerCodeId{code::power##V};        \
+    constexpr auto kStorageMarketCodeId{code::market##V};      \
+    constexpr auto kStorageMinerCodeId{code::miner##V};        \
+    constexpr auto kMultisigCodeId{code::multisig##V};         \
+    constexpr auto kInitCodeId{code::init##V};                 \
+    constexpr auto kPaymentChannelCodeId{code::paych##V};      \
+    constexpr auto kRewardActorCodeId{code::reward##V};        \
+    constexpr auto kSystemActorCodeId{code::system##V};        \
+    constexpr auto kVerifiedRegistryCodeId{code::verifreg##V}; \
+  }
+VM_ACTOR_BUILTIN_V_CODE(0)
+VM_ACTOR_BUILTIN_V_CODE(2)
+VM_ACTOR_BUILTIN_V_CODE(3)
+VM_ACTOR_BUILTIN_V_CODE(4)
```

Comment:
Is it time to add v5 now?

Reply:
Yes, in other pr (with actor/network version and others)